CMakeLists.txt

```diff
@@ -255,6 +255,27 @@ endif()
 # deployable code which need to specify the proper target chip.
 #-------------------------------------------------------------------------------
 
+# When ROCm is detected, force the user to make a conscious decision to enable
+# or disable ROCm testing. It's hard to enable automatically for a given target
+# chip because that would rely too much on guessing what the target chip is.
+# But silently continuing with ROCm testing disabled by default has also been
+# surprising.
+if (NOT DEFINED CACHE{IREE_HIP_TEST_TARGET_CHIP})
```

Comment:
Shouldn't we either make this conditional on the hip backend being enabled or also warn when the backend is not being built?